BoneReconstructionPlanner/BoneReconstructionPlanner.py

```diff
@@ -451,16 +465,8 @@ def __init__(self):
     Called when the logic class is instantiated. Can be used for initializing member variables.
     """
     ScriptedLoadableModuleLogic.__init__(self)
-    self.fibulaPlanesFolder = 0
-    self.mandible2FibulaTransformsFolder = 0
-    self.planeCutsFolder = 0
-    self.cutBonesFolder = 0
-    self.bonePiecesTransformFolder = 0
-    self.segmentationModelsFolder = 0
-    self.cutBonesList = []
-    self.fibulaPlanesList = []
-    self.planeCutsList = []
     self.rotTransformParameters = []
+    self.rotTransformParameters2 = []

```

Comment:
Need to use a much better name than rotTransformParameters2!
And of course specify it in the design.pptx.

Reply:
I think these variables won't be necessary after I remake the rotation transforms